src/dataIntegration/transcriptomics/preprocessing/selectGeneFromGPR.m

```diff
@@ -1,4 +1,4 @@
-function [expressionCol, gene_used] = selectGeneFromGPR(model, gene_names, gene_exp, parsedGPR, minSum)
+function expressionRxns = selectGeneFromGPR(model, gene_names, gene_exp, gene_sig, parsedGPR, minSum)
```

Comment:
Please refactor this so that gene_sig is an optional argument, or implement it thus that the old signature can still be used.
Since the number of optional arguments starts to increase, I would suggest to modify the function to use parameter/value pairs while still accepting the original signature. For a potential way to do so, have a look at e.g. addReaction. 
Modifying the header of the function in the way you currently suggest would potentially break a lot of existing code. 

Also the current outputs should be retained. Also admittedly, they are currently lacking documentation.